src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -50,8 +56,8 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
         </View>
         <View style={styles.containerButton}>
           <Pressable style={styles.followAction(32, 32)}>
-            <Animated.View style={{position: 'absolute', opacity: followIconFadeAnimation}}>
-              <MemoIc_Checklist width={32} height={32} />
+            <Animated.View style={{position: 'absolute', top: 4, opacity: followIconFadeAnimation}}>
+              <MemoIc_Checklist width={32} height={32} color={isAnon ? '#107793' : '#4782D7'} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `MemoIc_Checklist` icon now depends on whether the user is anonymous. This change seems reasonable and doesn't appear to introduce any issues.



<!-- This is an auto-generated comment by OSS CodeRabbit -->